doc/proposals/rlp-target-gateway-resource.md

```diff
@@ -0,0 +1,327 @@
+# RLP can target a Gateway resource
```

Comment:
Well, this is more than a proposal. This is already what it is implemented. 

Should be included in the docs, not in the proposals.

Reply:
maybe we should leave it here for now and remove later in other PRs.